aten/src/ATen/native/mps/operations/Shape.mm

```diff
@@ -192,6 +192,9 @@ void check_shape_except_dim(const Tensor &first, const Tensor &second,
        MemoryFormat memory_format,
        const Tensor& out) {
   using namespace mps;
+
+  memory_format = MemoryFormat::Contiguous;
```

Comment:
How does this fix the issue? Does the test_mps pass with this change?
If you want memory_format to be contiguous, then why don't you pass the input tensor as contiguous?

Reply:
@razarmehr , this is similar to what Denis is doing in the case of elu_backward. We need to make sure the NHWC case is handled. I think we should centralize the contiguous call and any copy which needs to happen.

Reply:
It passes test_mps locally. The ACDC case passes after the fix.  I'll work on a unit test that is smaller enough to show the case.

Reply:
OK, so we're bypassing the NHWC fix that was committed before by explicitly making the tensors contiguous before doing cat. If you think this won't affect performance much, we can go with it to unblock us, and later come up with a proper fix.